unlock-app/src/components/interface/checkout/main/Confirm.tsx

```diff
@@ -85,8 +84,7 @@ export function Confirm({
   communication,
 }: Props) {
   const [state, send] = useActor(checkoutService)
-  const { account, network, isUnlockAccount, changeNetwork } = useAuth()
-  const walletService = useWalletService()
+  const { account, network, getWalletService } = useAuth()
```

Comment:
Are we using `network` ?

I think we should _remove_ the `network` from the `useAuth` hook completely because it is a code smell IMO. It means a component needs to know on what network the user is, which is probably not right.

Reply:
We still use it in some places. I think we can remove it eventually but to minimize changes, I'm keeping it as it is.